ccd-definition/Categories/Categories.json

```diff
@@ -1007,6 +1007,14 @@
     "DisplayOrder": "4",
     "ParentCategoryID": "orders"
   },
+  {
+    "LiveFrom": "05/12/2022",
+    "CaseTypeID": "CIVIL",
+    "CategoryID": "judgments",
+    "CategoryLabel": "Judgments",
+    "DisplayOrder": "6",
```

Comment:
Isn't DisplayOrder 5?

Reply:
In Categories-SettleDiscontinue-nonprod.json, DisplayOrder as 5 is already given to discontinueNotices.

Reply:
Shouldn't this go in a non-prod file?

Reply:
Done